observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,63 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="^1,^1,^1,^1,^1,^1", options="header"]
+|===
+| *COO Version*   | *OCP Versions*       | *Dashboards*   | *Distributed Tracing*   | *Logging*   | *Troubleshooting Panel* 
+
+| 0.2.0           | 4.11                 | ✔              | ✘                       | ✘           | ✘ 
+| 0.3.0+          | 4.11 - 4.15          | ✔              | ✔                       | ✔           | ✘ 
+| 0.3.0+          | 4.16+                | ✔              | ✔                       | ✔           | ✔ 
+|===
+
+
+[id="cluster-observability-operator-release-notes-0-4-0_{context}"]
+== {coo-full} 0.4.0
+
+The following advisory is available for {coo-full} 0.4.0:
+
+// Change the errata link after release and just before publishing
+* link://https://errata.devel.redhat.com/advisory/138050[138050 {coo-full} 0.4.0]
```

Comment:
I can't open this link. I'm on VPN and running Kerberos.
Also, if this link requires VPN or Kerberos, then can customers see it?